### ISSUE TITLE
2025-01-24, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#61]
# Today's Arxiv Papers

(1) [CRPO: Confidence-Reward Driven Preference Optimization for Machine Translation](http://arxiv.org/abs/2501.13927v1), 2025-01-23

(2) [The Breeze 2 Herd of Models: Traditional Chinese LLMs Based on Llama with Vision-Aware and Function-Calling Capabilities](http://arxiv.org/abs/2501.13921v1), 2025-01-23

(3) [Analysis of Indic Language Capabilities in LLMs](http://arxiv.org/abs/2501.13912v1), 2025-01-23

(4) [Privacy-Preserving Personalized Federated Prompt Learning for Multimodal Large Language Models](http://arxiv.org/abs/2501.13904v1), 2025-01-23

(5) [Exploring Finetuned Audio-LLM on Heart Murmur Features](http://arxiv.org/abs/2501.13884v1), 2025-01-23

(6) [The machine learning platform for developers of large systems](http://arxiv.org/abs/2501.13881v1), 2025-01-23

(7) [A RAG-Based Institutional Assistant](http://arxiv.org/abs/2501.13880v1), 2025-01-23

(8) [On the Reasoning Capacity of AI Models and How to Quantify It](http://arxiv.org/abs/2501.13833v1), 2025-01-23

(9) [Predicting Compact Phrasal Rewrites with Large Language Models for ASR Post Editing](http://arxiv.org/abs/2501.13831v1), 2025-01-23

(10) [Hallucinations Can Improve Large Language Models in Drug Discovery](http://arxiv.org/abs/2501.13824v1), 2025-01-23

(11) [Large Language Model driven Policy Exploration for Recommender Systems](http://arxiv.org/abs/2501.13816v1), 2025-01-23

(12) [Enhancing LLMs for Governance with Human Oversight: Evaluating and Aligning LLMs on Expert Classification of Climate Misinformation for Detecting False or Misleading Claims about Climate Change](http://arxiv.org/abs/2501.13802v1), 2025-01-23

(13) [Not Every AI Problem is a Data Problem: We Should Be Intentional About Data Scaling](http://arxiv.org/abs/2501.13779v1), 2025-01-23

(14) [Explainable XR: Understanding User Behaviors of XR Environments using LLM-assisted Analytics Framework](http://arxiv.org/abs/2501.13778v1), 2025-01-23

(15) [Do Large Language Models Truly Understand Geometric Structures?](http://arxiv.org/abs/2501.13773v1), 2025-01-23

(16) [Tune In, Act Up: Exploring the Impact of Audio Modality-Specific Edits on Large Audio Language Models in Jailbreak](http://arxiv.org/abs/2501.13772v1), 2025-01-23

(17) [UGMathBench: A Diverse and Dynamic Benchmark for Undergraduate-Level Mathematical Reasoning with Large Language Models](http://arxiv.org/abs/2501.13766v1), 2025-01-23

(18) [EICopilot: Search and Explore Enterprise Information over Large-scale Knowledge Graphs with LLM-driven Agents](http://arxiv.org/abs/2501.13746v1), 2025-01-23

(19) [GPT-HTree: A Decision Tree Framework Integrating Hierarchical Clustering and Large Language Models for Explainable Classification](http://arxiv.org/abs/2501.13743v1), 2025-01-23

(20) [An Empirical Study of Retrieval-Augmented Code Generation: Challenges and Opportunities](http://arxiv.org/abs/2501.13742v1), 2025-01-23

(21) [Pseudocode-Injection Magic: Enabling LLMs to Tackle Graph Computational Tasks](http://arxiv.org/abs/2501.13731v1), 2025-01-23

(22) [RPO: Retrieval Preference Optimization for Robust Retrieval-Augmented Generation](http://arxiv.org/abs/2501.13726v1), 2025-01-23

(23) [Musical ethnocentrism in Large Language Models](http://arxiv.org/abs/2501.13720v1), 2025-01-23

(24) [EventVL: Understand Event Streams via Multimodal Large Language Model](http://arxiv.org/abs/2501.13707v1), 2025-01-23

(25) [DI-BENCH: Benchmarking Large Language Models on Dependency Inference with Testable Repositories at Scale](http://arxiv.org/abs/2501.13699v1), 2025-01-23

(26) [Question Answering on Patient Medical Records with Private Fine-Tuned LLMs](http://arxiv.org/abs/2501.13687v1), 2025-01-23

(27) [HumorReject: Decoupling LLM Safety from Refusal Prefix via A Little Humor](http://arxiv.org/abs/2501.13677v1), 2025-01-23

(28) [How to Complete Domain Tuning while Keeping General Ability in LLM: Adaptive Layer-wise and Element-wise Regularization](http://arxiv.org/abs/2501.13669v1), 2025-01-23

(29) [LVPruning: An Effective yet Simple Language-Guided Vision Token Pruning Approach for Multi-modal Large Language Models](http://arxiv.org/abs/2501.13652v1), 2025-01-23

(30) [Sigma: Differential Rescaling of Query, Key and Value for Efficient Language Models](http://arxiv.org/abs/2501.13629v1), 2025-01-23

(31) [FedPref: Federated Learning Across Heterogeneous Multi-objective Preferences](http://dx.doi.org/10.1145/3708984), 2025-01-23

(32) [Text-to-SQL based on Large Language Models and Database Keyword Search](http://arxiv.org/abs/2501.13594v1), 2025-01-23

(33) [Improving Contextual Faithfulness of Large Language Models via Retrieval Heads-Induced Optimization](http://arxiv.org/abs/2501.13573v1), 2025-01-23

(34) [LLMs Can Plan Only If We Tell Them](http://arxiv.org/abs/2501.13545v1), 2025-01-23

(35) [ReasVQA: Advancing VideoQA with Imperfect Reasoning Process](http://arxiv.org/abs/2501.13536v1), 2025-01-23

(36) [RECALL: Library-Like Behavior In Language Models is Enhanced by Self-Referencing Causal Cycles](http://arxiv.org/abs/2501.13491v1), 2025-01-23

(37) [Adaptive Testing for LLM-Based Applications: A Diversity-based Approach](http://arxiv.org/abs/2501.13480v1), 2025-01-23

(38) [Streaming Video Understanding and Multi-round Interaction with Memory-enhanced Knowledge](http://arxiv.org/abs/2501.13468v1), 2025-01-23

(39) [Spurious Forgetting in Continual Learning of Language Models](http://arxiv.org/abs/2501.13453v1), 2025-01-23

(40) [MultiDreamer3D: Multi-concept 3D Customization with Concept-Aware Diffusion Guidance](http://arxiv.org/abs/2501.13449v1), 2025-01-23

(41) [Softplus Attention with Re-weighting Boosts Length Extrapolation in Large Language Models](http://arxiv.org/abs/2501.13428v1), 2025-01-23

(42) [VulnBot: Autonomous Penetration Testing for A Multi-Agent Collaborative Framework](http://arxiv.org/abs/2501.13411v1), 2025-01-23

(43) [Can Large Language Models Understand Preferences in Personalized Recommendation?](http://arxiv.org/abs/2501.13391v1), 2025-01-23

(44) [Do as We Do, Not as You Think: the Conformity of Large Language Models](http://arxiv.org/abs/2501.13381v1), 2025-01-23

(45) [50 Shades of Deceptive Patterns: A Unified Taxonomy, Multimodal Detection, and Security Implications](http://arxiv.org/abs/2501.13351v1), 2025-01-23

(46) [Full-Stack Optimized Large Language Models for Lifelong Sequential Behavior Comprehension in Recommendation](http://arxiv.org/abs/2501.13344v1), 2025-01-23

(47) [Multi-aspect Knowledge Distillation with Large Language Model](http://arxiv.org/abs/2501.13341v1), 2025-01-23

(48) [AgentRec: Agent Recommendation Using Sentence Embeddings Aligned to Human Feedback](http://arxiv.org/abs/2501.13333v1), 2025-01-23

(49) [Qrazor: Reliable and effortless 4-bit llm quantization by significant data razoring](http://arxiv.org/abs/2501.13331v1), 2025-01-23

(50) [SplitLLM: Hierarchical Split Learning for Large Language Model over Wireless Network](http://arxiv.org/abs/2501.13318v1), 2025-01-23

(51) [Representing Visualization Insights as a Dense Insight Network](http://arxiv.org/abs/2501.13309v1), 2025-01-23

(52) [OSUM: Advancing Open Speech Understanding Models with Limited Resources in Academia](http://arxiv.org/abs/2501.13306v1), 2025-01-23

(53) [Watching the AI Watchdogs: A Fairness and Robustness Analysis of AI Safety Moderation Classifiers](http://arxiv.org/abs/2501.13302v1), 2025-01-23

(54) [Hypothesis Generation for Materials Discovery and Design Using Goal-Driven and Constraint-Guided LLM Agents](http://arxiv.org/abs/2501.13299v1), 2025-01-23

(55) [RAMQA: A Unified Framework for Retrieval-Augmented Multi-Modal Question Answering](http://arxiv.org/abs/2501.13297v1), 2025-01-23

(56) [Toyteller: AI-powered Visual Storytelling Through Toy-Playing with Character Symbols](http://dx.doi.org/10.1145/3706598.3713435), 2025-01-23

(57) [RAG-Reward: Optimizing RAG with Reward Modeling and RLHF](http://arxiv.org/abs/2501.13264v1), 2025-01-22

(58) [Exploring GPT's Ability as a Judge in Music Understanding](http://arxiv.org/abs/2501.13261v1), 2025-01-22

(59) [Bypassing Array Canaries via Autonomous Function Call Resolution](http://arxiv.org/abs/2501.13256v1), 2025-01-22

(60) [VideoLLaMA 3: Frontier Multimodal Foundation Models for Image and Video Understanding](http://arxiv.org/abs/2501.13106v2), 2025-01-22

(61) [Refining Input Guardrails: Enhancing LLM-as-a-Judge Efficiency Through Chain-of-Thought Fine-Tuning and Alignment](http://arxiv.org/abs/2501.13080v1), 2025-01-22

(62) [Does Table Source Matter? Benchmarking and Improving Multimodal Scientific Table Understanding and Reasoning](http://arxiv.org/abs/2501.13042v1), 2025-01-22

(63) [MONA: Myopic Optimization with Non-myopic Approval Can Mitigate Multi-step Reward Hacking](http://arxiv.org/abs/2501.13011v1), 2025-01-22

(64) [Pairwise RM: Perform Best-of-N Sampling with Knockout Tournament](http://arxiv.org/abs/2501.13007v1), 2025-01-22

(65) [Large Language Model-Based Semantic Communication System for Image Transmission](http://arxiv.org/abs/2501.12988v1), 2025-01-22

(66) [LLM4WM: Adapting LLM for Wireless Multi-Tasking](http://arxiv.org/abs/2501.12983v1), 2025-01-22

(67) [Implicit Causality-biases in humans and LLMs as a tool for benchmarking LLM discourse capabilities](http://arxiv.org/abs/2501.12980v1), 2025-01-22

(68) [OnionEval: An Unified Evaluation of Fact-conflicting Hallucination for Small-Large Language Models](http://arxiv.org/abs/2501.12975v1), 2025-01-22

(69) [Accessible Smart Contracts Verification: Synthesizing Formal Models with Tamed LLMs](http://arxiv.org/abs/2501.12972v1), 2025-01-22

(70) [It's complicated. The relationship of algorithmic fairness and non-discrimination regulations in the EU AI Act](http://arxiv.org/abs/2501.12962v1), 2025-01-22

(71) [Efficient Prompt Compression with Evaluator Heads for Long-Context Transformer Inference](http://arxiv.org/abs/2501.12959v1), 2025-01-22

(72) [GANQ: GPU-Adaptive Non-Uniform Quantization for Large Language Models](http://arxiv.org/abs/2501.12956v1), 2025-01-22

(73) [DeepSeek-R1: Incentivizing Reasoning Capability in LLMs via Reinforcement Learning](http://arxiv.org/abs/2501.12948v1), 2025-01-22

(74) [Correctness Assessment of Code Generated by Large Language Models Using Internal Representations](http://arxiv.org/abs/2501.12934v1), 2025-01-22

(75) [FilmAgent: A Multi-Agent Framework for End-to-End Film Automation in Virtual 3D Spaces](http://arxiv.org/abs/2501.12909v1), 2025-01-22

(76) [A Functional Software Reference Architecture for LLM-Integrated Systems](http://arxiv.org/abs/2501.12904v1), 2025-01-22

(77) [Architectural Fusion Through Contextual Partitioning in Large Language Models: A Novel Approach to Parameterized Knowledge Integration](http://arxiv.org/abs/2501.12901v1), 2025-01-22

(78) [Test-Time Preference Optimization: On-the-Fly Alignment via Iterative Textual Feedback](http://arxiv.org/abs/2501.12895v1), 2025-01-22

(79) [Generative AI Misuse Potential in Cyber Security Education: A Case Study of a UK Degree Program](http://arxiv.org/abs/2501.12883v2), 2025-01-22

(80) [WisdomBot: Tuning Large Language Models with Artificial Intelligence Knowledge](http://arxiv.org/abs/2501.12877v1), 2025-01-22

(81) [Mutation-Guided LLM-based Test Generation at Meta](http://arxiv.org/abs/2501.12862v1), 2025-01-22

(82) [ACEBench: Who Wins the Match Point in Tool Learning?](http://arxiv.org/abs/2501.12851v1), 2025-01-22

(83) [Adaptive Retrieval Without Self-Knowledge? Bringing Uncertainty Back Home](http://arxiv.org/abs/2501.12835v1), 2025-01-22

(84) [Open or Closed LLM for Lesser-Resourced Languages? Lessons from Greek](http://arxiv.org/abs/2501.12826v1), 2025-01-22

(85) [Revisit Self-Debugging with Self-Generated Tests for Code Generation](http://arxiv.org/abs/2501.12793v1), 2025-01-22

(86) [Generating Diverse Q&A Benchmarks for RAG Evaluation with DataMorgana](http://arxiv.org/abs/2501.12789v1), 2025-01-22

(87) [LLMs as Repositories of Factual Knowledge: Limitations and Solutions](http://arxiv.org/abs/2501.12774v1), 2025-01-22

(88) [NExtLong: Toward Effective Long-Context Training without Long Documents](http://arxiv.org/abs/2501.12766v1), 2025-01-22

(89) [Intelligent Exercise and Feedback System for Social Healthcare using LLMOps](http://arxiv.org/abs/2501.13723v1), 2025-01-22

(90) [EvidenceMap: Unleashing the Power of Small Language Models with Evidence Analysis for Biomedical Question Answering](http://arxiv.org/abs/2501.12746v1), 2025-01-22

(91) [Online Preference Alignment for Language Models via Count-based Exploration](http://arxiv.org/abs/2501.12735v1), 2025-01-22

(92) [A Call for Critically Rethinking and Reforming Data Analysis in Empirical Software Engineering](http://arxiv.org/abs/2501.12728v1), 2025-01-22

(93) [Paradigm-Based Automatic HDL Code Generation Using LLMs](http://arxiv.org/abs/2501.12702v1), 2025-01-22

(94) [Training Dialogue Systems by AI Feedback for Improving Overall Dialogue Impression](http://arxiv.org/abs/2501.12698v1), 2025-01-22

(95) [Combining Knowledge Graph and LLMs for Enhanced Zero-shot Visual Question Answering](http://arxiv.org/abs/2501.12697v1), 2025-01-22

(96) [EchoLM: Accelerating LLM Serving with Real-time Knowledge Distillation](http://arxiv.org/abs/2501.12689v1), 2025-01-22

(97) [Training Data Attribution (TDA): Examining Its Adoption & Use Cases](http://arxiv.org/abs/2501.12642v1), 2025-01-22

(98) [SoMa: Identifying, Exploring, and Understanding the DRAM Communication Scheduling Space for DNN Accelerators](http://arxiv.org/abs/2501.12634v1), 2025-01-22

(99) [Distillation Quantification for Large Language Models](http://arxiv.org/abs/2501.12619v1), 2025-01-22

(100) [Deep Learning-Based Identification of Inconsistent Method Names: How Far Are We?](http://dx.doi.org/10.1007/s10664-024-10592-z), 2025-01-22
